tools/terraform_sync_tool/README.md

```diff
@@ -0,0 +1,72 @@
+# Terraform Sync Tool
+
+This directory contains the setup for the Terraform Sync Tool. Terraform Sync Tool was designed to address the schema drifts in BigQuery tables and keep the 
+Terraform schemas up-to-date with the BigQuery table schemas in production environment.
+
+Terraform Sync Tool can be integrated into your CI/CD pipeline using Cloud Build. You'll need to add two steps to `cloudbuild.yaml`. 
+- Step 0: Use Terragrunt command to detect resource drifts and write output into a JSON file
+- Step 1: Use Python scripts to identify and investigate the drifts
+
+Cloud Build fails the build attemps if resource drifts are detected and notifies the latest resource information. Developers should be able to update
+the Terraform resources accordingly. 
+
+## Prerequisite
+Before building the terraform sync tool, please ensure that billing and Cloud Build are enabled for your Cloud project.
+
+You'll also need to install Cloud SDK(https://cloud.google.com/sdk/docs/install) and Terragrunt(https://terragrunt.gruntwork.io/docs/getting-started/install)
```

Comment:
Cloud SDK is not a necessary requirement

Reply:
README Updated